core/lib/groups/TensorProduct.agda

```diff
@@ -43,11 +43,37 @@ module TensorProduct₀ {i} {j} (G : AbGroup i) (H : AbGroup j) where
     _⊗_ : G.El → H.El → El
     _⊗_ g h = insert (g , h)
 
-    lin-l : ∀ g₁ g₂ h → G.comp g₁ g₂ ⊗ h == comp (g₁ ⊗ h) (g₂ ⊗ h)
-    lin-l g₁ g₂ h = rel-holds (linear-l g₁ g₂ h)
-
-    lin-r : ∀ g h₁ h₂ → g ⊗ H.comp h₁ h₂ == comp (g ⊗ h₁) (g ⊗ h₂)
-    lin-r g h₁ h₂ = rel-holds (linear-r g h₁ h₂)
+    ⊗-lin-l : ∀ g₁ g₂ h → G.comp g₁ g₂ ⊗ h == comp (g₁ ⊗ h) (g₂ ⊗ h)
+    ⊗-lin-l g₁ g₂ h = rel-holds (linear-l g₁ g₂ h)
+
+    ⊗-lin-r : ∀ g h₁ h₂ → g ⊗ H.comp h₁ h₂ == comp (g ⊗ h₁) (g ⊗ h₂)
+    ⊗-lin-r g h₁ h₂ = rel-holds (linear-r g h₁ h₂)
+
+    ⊗-ident-l : ∀ h → G.ident ⊗ h == ident
+    ⊗-ident-l h = cancel-l (G.ident ⊗ h) $
```

Comment:
I believe we name this lemma as `unit-l` in other places.

Reply:
That's right. I will change that.